audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/ServiceName.java

```diff
@@ -22,10 +22,21 @@
  * @since 0.4.0
  */
 public enum ServiceName {
-    AUDIT_SERVICE,
```

Comment:
If updating list of services, then update the @since tag in the javadoc to 0.5.0

Reply:
Updated the Since tag